omero/developers/Web/Webclient.txt

```diff
@@ -0,0 +1,82 @@
+OMERO.webclient
+===============
+
+The webclient Django app provides the main OMERO.web client UI.
+
+Data from OMERO is rendered to html using Django templates before being
+sent to the browser. Additional javascript (jQuery-based) functionality in the browser
+is used to update the UI in response to user interactions such as 
+browsing or modifying the data.
+
+
+Top level pages
+---------------
+
+There are a small number of top level html pages that the user will start at.
+These are all handled by the load_template view:
+
+ - / (homepage delegates to /userdata)
```

Comment:
These bullet points shouldn't be indented, it's causing that annoying purple highlighting in plone.